### PR TITLE
Notifications full flow

### PR DESCRIPTION
**Pull request by milenakrawczyk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,16 +1,95 @@
...
+    } catch (err) {
+      throw Error(err);
+    }
+  };
+
+const getSubscription = async (pool, accountId) => {
+    try {
+      return await pool('Subscription')
+      .select('push_subscription_object')
+      .where('account', accountId)
```
Comment (charleslavon) [inline index.js:55]: @milenakrawczyk is the expectation that there will only be 1 subscription per user?

Comment (milenakrawczyk) [reply index.js:55]: Yes, I wanted to implement the simplest flow for now just to test and see how it behaves but I may extend it to encompass multiple subscriptions per user. What would be you preference?

Comment (charleslavon) [reply index.js:55]: We should have 1:many account-to-push subscriptions for the MVP

File: index.js [hunk 2]
```diff
@@ -1,16 +1,95 @@
 const functions = require('@google-cloud/functions-framework');
-const axios = require("axios");
+const webpush = require('web-push');
+const Knex = require('knex');
 
+webpush.setVapidDetails(
+  process.env.VAPID_SUBJECT || 'mailto:milena@near.org',
```
Comment (charleslavon) [inline index.js:6]: For a future PR, a helpful alternative would be to set defaults in a local properties file

**Review by charleslavon — APPROVED** [review]

**Pull request merged**